spec/index.html

```diff
@@ -706,7 +706,25 @@ <h3>Properties of simple entailment (Informative)</h3>
 
     <p class="fact"> If E contains an IRI which does not occur anywhere in S,
       then S does not simply entail E.</p>
+      
+    <p>The following semantic properties relate triple terms and triples asserted in a graph, and they introduce a general definition of satisfiability.</p>
+    
+    <p>We define the <dfn>set of propositions</dfn> in an interpretation as follows:</p>
+
+	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;&lt;x, y, z&gt;&#65372;x is in IR, y is in IP, z is in IR&nbsp;}; we observe that the propositions are exactly the domain of the RE mapping used to denote triple terms as resources. </p>
+
+	<p>We define the <dfn>set of facts</dfn> in an interpretation as follows:</p>
+
+	<p class="fact"> The set F of facts in an interpretation I is F(I) = {&nbsp;&lt;x, y, z&gt;&#65372;&lt;x, z&gt; is in IEXT(y)&nbsp;}; it is easy to see that the facts are the propositions which are true in the interpretation. </p>
+
+	<p>Given a blank node mapping, we define the <dfn>set of facts asserted by a graph</dfn> in an interpretation as follows:</p>
+
+	<p class="fact">Given a blank node mapping A, the set of all facts asserted by a graph G in an interpretation I is FEXT(G, I, A) = {&nbsp;&lt;&nbsp;[I+A](s), I(p), [I+A](o)&nbsp;&gt;&#65372;`s p o.` is in G&nbsp;}; we observe that given a blank node mapping, the asserted facts of a graph in an interpretation may not necessarily be among the facts of the interpretation.</p>
```

Comment:
```suggestion
	<p class="fact">Given a blank node mapping A, the set of all facts asserted by a graph G in an interpretation I is FEXT(G, I, A) = {&nbsp;&lt;&nbsp;[I+A](s), I(p), [I+A](o)&nbsp;&gt;&#65372;`s p o.` is in G&nbsp;}; we observe that given a blank node mapping, the asserted facts of a graph with respect to an interpretation may not necessarily be among the facts of the interpretation.</p>
```
I prefer this kind of wording. I don't see the graph doing anything **in** the interpretation.

Reply:
Fixed.